lib/simp/cli/utils.rb

```diff
@@ -98,31 +137,47 @@ def generate_password(length = DEFAULT_PASSWORD_LENGTH )
 
   # Add custom facts to the Facter search path and then reload the facts
   #
-  # +module_paths+: Array of module paths in which to find custom facts
-  # +add_to_env+:   Whether to add the custom fact paths to the FACTERLIB
-  #   environment variable, in addition, so that the facts available to
-  #   any spawned processes that use FACTERLIB, (e.g. `puppet apply`)
-  def load_custom_facts(module_paths, add_to_env = false)
+  # This can be called as many times as necessary and will re-load the facts
+  # each time with the new path.
+  #
+  # @param module_paths [Array[String]]
+  #   Paths to search for custom facts
+  # @param add_to_env [Boolean]
+  #   Whether to add the custom fact paths to the FACTERLIB environment
+  #   variable, in addition, so that the facts available to any spawned
+  #   processes that use FACTERLIB, (e.g. `puppet apply`)
+  def load_custom_facts(module_paths=[], add_to_env = false)
```

Comment:
I'm going to defer on this one for now. I mean, it's literally a `Find` that dumps items into the load path. If we note a regression at some point, then we should definitely add a test but, for now, I think that it's more work that it would be worth in the long run.